src/test/groovy/graphql/util/FpKitTest.groovy

```diff
@@ -98,6 +98,100 @@ class FpKitTest extends Specification {
         l == ["Parrot"]
     }
 
+    class Person {
```

Comment:
hey look -  TESTS!

We never had very many tests for FpKit

So I I write them first - then changed the implementation

TDD!!!

Reply:
Noice